doc/usermanual/lighttable/panels/export.xml

```diff
@@ -790,6 +807,79 @@
+        synonyms and omit to include the path elements of hierarchical tags. Remember that a tag set up as a category is never exported.
+      </para>
+
+      <para>
+        The right part lets you define a formula per metadata. What you define here
+        has priority on the left settings.
+        The first column identifies the metadata. The second one is editable and defines the metadata calculation.
+        Letting the formula empty prevents to export this specific metadata (Exif.GPSInfo.GPSVersionID in the above example).
+        Otherwise you can use any of the variables and rules defined above for filename.
+        Click on <emphasis>enter</emphasis> to validate the formula.
+      </para>
+
+      <para>
+        Use the icon <quote>-</quote> to remove a metadata from the list, the
+        icon <quote>+</quote> to add a new one from the predefined list:
+        <graphic fileref="lighttable/images/panel_exportselected-metadata-selection.png" scalefit="1" width="40%" align="center" />
+        Double-click or click on button <emphasis>add</emphasis> to add a metadata to the configuration list.
+      </para>
+
+      <para>
+        The formulas allow you to get the most of the tagging function. You can virtually define
+        all the metadata you need to qualify your images in tagging and, thanks to these
+        formulas, export the values in the xmp or iptc tags of your choice. The exported tags
+        can be different from one export to the other depending on the destination of the images.
+        Just remember that tags set as <emphasis>category</emphasis> are not exported.
+        Tags and Categories are displayed separately in image informations.
+      </para>
+
+      <para>
+        Example 1. A first level tag called <emphasis>places</emphasis> set as category followed
+        by four levels, country, region, city and location: <code>places|France|Nord|Lille|rue Nationale</code>.
+        The formulas <code>$(CATEGORY0(places))</code>, <code>$(CATEGORY1(places))</code>,
+        <code>$(CATEGORY2(places))</code> and <code>$(CATEGORY3(places))</code>
+        retrieve the four values (when exist) which can be assigned separately to the xmp tags you need.
+        Image informations display the four values in tags list. They can also be exported as simple tags <code>$(TAGS)</code>, unlike
+        the tag <emphasis>places</emphasis>.
+      </para>
+
+      <para>
+        Example 2. A first level tag called <emphasis>creator</emphasis> followed by the name of the photographer,
+        both set as <emphasis>category</emphasis>: <code>creator|firstname lastname</code>.
+        The formula <code>copyrights $(YEAR) $(CATEGORY0(creator))</code> builds the text associated to image rights.
+        Image informations display in categories the following: <code>creator: firstname lastname</code>.
+        <code>firstname lastname</code> doesn't appear in tags list and is not exported as simple tag.
```

Comment:
I would say that "`neither <code>creator</code> nor <code>firstname lastname</code> appear in tags list; they are also not exported as simple tags`".

Reply:
yes, you got it. :)